docs/analysis/index.rst

```diff
@@ -235,22 +210,6 @@ is stored in the ``flux_points`` property as a `~gammapy.spectrum.FluxPoints` ob
           8.79922543569107  1.877442373267013e-13  7.097738087032472e-14  1.254638299336029e-13 2.2705519890120373e-14 False
     >>> analysis.flux_points.peek()
 
-Residuals
----------
-
-For 3D analysis we can compute a residual image to check how good are the models
-for the source and/or the background.
-
-.. code-block:: python
-
-    >>> analysis.datasets.datasets[0].residuals()
```

Comment:
Why remove this example how to get residuals?
OK to remove if you don't think it's good to have at this place.
If it's just because it was broken, I think this would work:
```
analysis.datasets[0].residuals()
analysis.datasets[0].plot_residuals()
```

Reply:
Yes, it was broken when I tested. It works when rebasing.
I've put it back.